rs/nns/governance/tests/governance.rs

```diff
@@ -14151,15 +14154,20 @@ fn test_neuron_info_private_enforcement() {
         ];
         main(neuron_id_to_expect_redact);
 
-        // Insepct visibility. This time, it should always be None.
-        for neuron_id in [
-            no_explicit_visibility_neuron.id.unwrap(),
-            private_neuron.id.unwrap(),
-            public_neuron.id.unwrap(),
-            known_neuron.id.unwrap(),
+        // Insepct visibility. This time, the no explicit visibility guy is shown as such.
```

Comment:
Did you mean to write "guy"?

Reply:
I think so, but maybe it would make more sense to say neuron.